### PR TITLE
Integration of EDA, requirements.txt, readme.md and additional material

### PR DESCRIPTION
**Pull request by auringonnousu**
Please check before you submit a PR:

- [x] Have you checked to ensure there aren't other open PR for the same update/change?
- [x] Have you read the CONTRIBUTING document?
- [x] Does the commit message follow our guidelines?
- [ ] Have you done your changes in a separate branch?
- [x] Give a descriptive title to you PR.
- [x] Give a description of your changes.
- [x] Other information:

Integration of EDA, requirements.txt, readme.md and additional material (images and text) into:

supervised learning

### CODE REVIEW

File: supervised_learning/images_text/readme.md
Comment (benjwolff) [inline supervised_learning/images_text/readme.md:1]: Please rename file name to README.md

File: supervised_learning/images_text/img/skewness.svg.licence
Comment (benjwolff) [inline supervised_learning/images_text/img/skewness.svg.licence:1]: Please rename to skewness.svg.license

File: supervised_learning/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# requirements.txt
```
Comment (benjwolff) [inline supervised_learning/requirements.txt]: This line is not necessary. Replace with something useful or remove.

File: supervised_learning/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+numpy==1.23.5
+pandas>=1.5.2
+matplotlib>=3.6.2
+openml>=0.13.0
+scikit-learn>=1.2.0
+seaborn>=0.12.2
+imblearn=>0.10.1
+
+# python==3.9.13
```
Comment (benjwolff) [inline supervised_learning/requirements.txt]: This line is not necessary. Please remove.

File: supervised_learning/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+#################################################################################
+# NOTE:
+
+# If you start a project from scratch (in a new environment), you can also install all modules in this requirements.txt via the command: pip install -r requirements.txt
+
+# This list has to be maintained in the future. For this, please check with: pip list --outdated
+
+# To check your installed version, use the command (in Jupyter Notebook): print('The installed version is {}.'.format(gseaborn.__version__))
+
+# For alternatives, go to https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html
```
Comment (benjwolff) [inline supervised_learning/requirements.txt]: Better remove this comment and move these instructions to your README.md in your root folder.

File: supervised_learning/requirements.txt [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# requirements.txt
+
+numpy==1.23.5
```
Comment (benjwolff) [inline supervised_learning/requirements.txt]: Is there any reason to pin this version to 1.23.5? I would either suggest setting a minimum version for every package or pinning everything. If you mix both, the dependencies will break in the future.

**Comment by auringonnousu** [comment]
fixed according to comments

**Pull request merged**